EWL-4429 Fixes typography for Promo as inline for News Article

designerneil opened the pull request:
Margin around the block

<!-- NOTE: Please just put "N/A" for any section below that isn't applicable to the work you've done, do not omit entirely. -->

## Ticket(s)

Please do not submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Jira Ticket**

- [EWL-4429: Fix typography and padding](https://issues.ama-assn.org/browse/EWL-4429)


## Description

```
Nick Aleck added a comment - 3 days ago
Topic Promo –

body font should be 400 weight
there needs to be increased padding around the promo (Viewing in Chrome)
```

Addresses the margin and typography weight

## To Test

- [ ] http://localhost:3000/?p=molecules-inline-block-copy 
- [ ] Observe font weight
- [ ] http://localhost:3000/?p=pages-news
- [ ] Observe margin around the inline block content

## Visual Regressions

none

## Relevant Screenshots/GIFs

<img width="817" alt="screen shot 2018-01-26 at 2 10 44 pm" src="https://user-images.githubusercontent.com/2271747/35458637-b91c350c-02a2-11e8-9e77-0c9dee2b4986.png">

## Remaining Tasks

Remaining tasks?


## Additional Notes

Anything more to add? Good things to call out here are:
- are there any PRs in this or other repositories that relate to or affect this PR?
- are there any caveats or oddities testers should know about when testing this PR?
- are there any problems you ran into during your work that you'd like to call out?

csj1012 committed:
[EWL-4388] added pattern files

csj1012 committed:
[EWL-4388] added pattern files

csj1012 committed:
[EWL-4388] added pattern files

csj1012 committed:
[EWL-4388] add pattern to the Topic page pattern and start populating .json data

csj1012 committed:
[EWL-4388] add pattern to the Topic page pattern and start populating .json data

csj1012 committed:
[EWL-4388] add pattern to the Topic page pattern and start populating .json data

designerneil committed:
Adds article small variant to lead news

designerneil committed:
Adds article small variant to lead news

designerneil committed:
Merge branch 'feature/EWL-4388-migrate-lead-news-block' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4388-migrate-lead-news-block

designerneil committed:
Adds lead news markdown

designerneil committed:
Adds lead news markdown

designerneil committed:
Adjust organism so D8 can use it by using the extends functionality

Commit:
refs EWL-4431: set font weight to 400, and update button colors

designerneil committed:
removes old colors and fonts
updates all files that used them

designerneil committed:
removes old colors and fonts
updates all files that used them

designerneil committed:
Adds new reds and greens

designerneil committed:
Adds new reds and greens

designerneil committed:
Merge branch 'feature/EWL-4388-migrate-lead-news-block' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4388-migrate-lead-news-block

designerneil committed:
Fixes lead news per PR
- Adds related content article stub
- modifies title only
- modifies metadata to use a parent container
- changes some of the conditionals for article stub

designerneil committed:
Fixes typography
Margin around the block

designerneil committed:
Fixes typography
Margin around the block

designerneil committed:
Fixes typography
Margin around the block

designerneil committed:
Fixes typography
Margin around the block

designerneil committed:
Fixes typography
Margin around the block

designerneil committed:
Fixes typography
Margin around the block

designerneil committed:
Fixes type sizes
take away unnecessary parent selector in json

designerneil committed:
removes paragraph from json

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Adds link to body copy
Adds class to underline all links inside of p tags

designerneil committed:
Fixes type size for teaser

designerneil committed:
Adds social share buttons

designerneil committed:
Adds social share buttons

designerneil committed:
Adds social share buttons

designerneil committed:
Adds social share buttons

designerneil committed:
changes date block to use h3

designerneil committed:
Adjust bio image to have a max of 80px

designerneil committed:
tags now have border on top and button
tags uses flexbox instead of inline elements
margin and padding have been adjusted to give the tags more air

designerneil committed:
Adds learn more to tools block

Commit:
Merge pull request #70 from AmericanMedicalAssociation/bugfix/EWL-4419-date-atom

EWL-4419 changes date atom to use h3

Commit:
fix empty line

Commit:
Merge pull request #73 from AmericanMedicalAssociation/bugfix/EWL-4430-tools-list-footer

EWL-4430 Adds learn more to tools block

Commit:
Merge pull request #72 from AmericanMedicalAssociation/bugfix/EWL-4423-tag-atom

EWL-4423 Adjust tags styling and spacing

Commit:
Merge pull request #71 from AmericanMedicalAssociation/bugfix/EWL-4420-inline-bio

EWL-4420 Adjust bio image to have a max of 80px

Commit:
Merge pull request #68 from AmericanMedicalAssociation/bugfix/EWL-4433-Column-teaser

EWL-4433 Fixes type size for the column teaser to be smaller

labbydev commented on styleguide/source/assets/scss/00-base/_font-variables.scss:
```diff
@@ -3,9 +3,9 @@ $myriad-pro: "myriad-pro", "Helvetica", "Arial", "Open Sans", sans-serif;
-$font-weight-bold: 600;
+$font-weight-regular: 400;
+$font-weight-semibold: 600;
+$font-weight-bold: 900;
```
It looks like this also updated the font weights on the article stubs in the column teaser and the four up teaser, were those also intended to be updated?

designerneil replied:
yes. it seems like UX wanted regular type to be 400 and semibold 600. 100 font-weight was too light.

labbydev reviewed (changes requested):
@designerneil can you also update it so that the promo as inline is full width on mobile in order to match the wireframe?

designerneil committed:
tools clean and integration with D8

designerneil committed:
updates per PR suggestion

designerneil committed:
Address concerns made in PR
removes unused font mixin file

designerneil committed:
Merge branch 'bugfix/EWL-4424-Create-News-Article-Template' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into bugfix/EWL-4424-Create-News-Article-Template

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Adds margins in between paragraphs

designerneil committed:
Merge branch 'bugfix/EWL-4418-social-share' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into bugfix/EWL-4418-social-share

designerneil committed:
Adds empty line

designerneil committed:
Adds empty line

designerneil committed:
adds learn more tools footer

Commit:
Add empty line

Commit:
Merge pull request #74 from AmericanMedicalAssociation/feature/EWL-4165-tools

tools clean and integration with D8

labbydev commented:
@designerneil can you also update it so that the promo as inline is full width on mobile in order to match the wireframe?

Commit:
Add empty line

Commit:
Merge pull request #66 from AmericanMedicalAssociation/bugfix/EWL-4432-four-up-teaser

Fixes type sizes for Article Stub and Four Up Teaser

designerneil commented:
@labbydev where is mobile wireframe?

designerneil committed:
Merge branch 'bugfix/EWL-4429-promo-as-inline' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into bugfix/EWL-4429-promo-as-inline

labbydev commented:
@designerneil https://sketch.cloud/s/L0dLQ/p/news-article/article-mobile is the mobile wireframe. Using the arrows at the top where it says `2/3` you can switch between Mobile, Tablet and Desktop wireframes.

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Adds margins between paragraphs to match mockup
Adds margin to the left of the article to match mockup

designerneil committed:
Merge branch 'bugfix/EWL-4424-Create-News-Article-Template' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into bugfix/EWL-4424-Create-News-Article-Template

designerneil committed:
changes names of article-metadata to better reflect BEM standards for what the molecule is

designerneil committed:
Merge branch 'chore/EWL-4455-remove-old-typography-colors' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into chore/EWL-4455-remove-old-typography-colors

Commit:
Merge pull request #59 from AmericanMedicalAssociation/feature/EWL-4388-migrate-lead-news-block

[EWL-4388] Migrate Lead News Block

designerneil committed:
Makes page responsive by adding breakpoints to enable the page to go full width when in mobile view
Adds gulp-clean to package.json

designerneil committed:
Makes page responsive by adding breakpoints to enable the page to go full width when in mobile view
Adds gulp-clean to package.json

designerneil committed:
Makes page responsive by adding breakpoints to enable the page to go full width when in mobile view
Adds gulp-clean to package.json

designerneil committed:
Makes page responsive by adding breakpoints to enable the page to go full width when in mobile view
Adds gulp-clean to package.json

designerneil committed:
Makes page responsive by adding breakpoints to enable the page to go full width when in mobile view
Adds gulp-clean to package.json

designerneil commented:
adds responsive breakpoints to enable the page to go full width when in mobile view.

Commit:
Merge branch 'develop' into chore/EWL-4455-remove-old-typography-colors

Commit:
Merge pull request #64 from AmericanMedicalAssociation/chore/EWL-4455-remove-old-typography-colors

EWL-4455 removes old colors and fonts

designerneil committed:
Takes ama__tag out of ama__tags

designerneil committed:
Takes ama__tag out of ama__tags

designerneil committed:
Takes ama__tag out of ama__tags

designerneil committed:
Takes ama__tag out of ama__tags

designerneil committed:
Merge branch 'bugfix/EWL-4424-Create-News-Article-Template' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into bugfix/EWL-4424-Create-News-Article-Template

designerneil committed:
Merge branch 'bugfix/EWL-4424-Create-News-Article-Template' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into bugfix/EWL-4424-Create-News-Article-Template

designerneil committed:
Adds back array restrictor for fourUpTeaser

designerneil committed:
Adds back array restrictor for fourUpTeaser

designerneil committed:
Adds back array restrictor for fourUpTeaser

designerneil committed:
Merge branch 'bugfix/EWL-4424-Create-News-Article-Template' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into bugfix/EWL-4424-Create-News-Article-Template

designerneil committed:
Merge branch 'bugfix/EWL-4418-social-share' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into bugfix/EWL-4418-social-share

designerneil committed:
Adds missing icon files

designerneil committed:
Adds missing icon files

Commit:
Merge pull request #69 from AmericanMedicalAssociation/bugfix/EWL-4418-social-share

EWL-4418 Adds social share buttons to article header

labbydev commented:
@designerneil this is appearing as full-width always now.

![screen shot 2018-01-31 at 11 13 58 am](https://user-images.githubusercontent.com/1653723/35636783-e948ef58-0677-11e8-91ac-9ad81eeb828d.png)

It also looks like the markup may be a bit off

```
<div class="ama__inline-block-copy">
  <div class="ama__rule"></div>
  <div class="ama__inline-block-copy--content">
    <h2 class="ama__h2">Title Text</h2>
    <p class=""> A paragraph (from the Greek paragraphos, "to write beside" or "written beside") is a self-contained unit of a discourse in writing dealing with a particular point or idea. A paragraph consists of one or more sentences. Though not required by the syntax of any language, paragraphs are usually an expected part of formal writing, used to organize longer prose.</p>
    
  <a href="#" class="ama__button  ama__button--secondary" title="Sign Up">
      Sign up
  </a>
  </div>
  <div class="ama__rule"></div>
</div>
```

It looks like you put the breakpoints for `&___copy` which the class should probably just be `ama__inline-block-copy`. Also `ama__inline-block-copy--content` should be a sub-element and therefore should be `ama__inline-block-copy__content`.

designerneil committed:
update ribbon to use the ama__white-link so I don't have to put an a tag inside of a p tag

designerneil committed:
update ribbon to use the ama__white-link so I don't have to put an a tag inside of a p tag

designerneil committed:
Merge branch 'bugfix/EWL-4424-Create-News-Article-Template' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into bugfix/EWL-4424-Create-News-Article-Template

designerneil committed:
Updates the way links behave.
By default links have and underline
Links with classes of ama__link--blue, ama__link--purple etc have no underlines
Removes class p tag with an a tag inside of it

designerneil committed:
Merge branch 'bugfix/EWL-4429-promo-as-inline' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into bugfix/EWL-4429-promo-as-inline

designerneil committed:
Fixes &__copy class inside of ama__inline-block to use ama__inline-block-copy
Fixes responsive rendering of the inline block

designerneil committed:
Fixes &__copy class inside of ama__inline-block to use ama__inline-block-copy
Fixes responsive rendering of the inline block

designerneil committed:
Fixes &__copy class inside of ama__inline-block to use ama__inline-block-copy
Fixes responsive rendering of the inline block

designerneil committed:
Fixes &__copy class inside of ama__inline-block to use ama__inline-block-copy
Fixes responsive rendering of the inline block

designerneil commented:
Thanks @labbydev for your suggestions I've simplified the inline-block css to use ama__inline-block-copy--content as oppose to &__copy which didn't do anything.

labbydev commented on styleguide/source/_patterns/02-molecules/inline-block-copy.twig:
```diff
@@ -1,8 +1,8 @@
 {% set heading, paragraph, button = inlineBlockCopy.heading, inlineBlockCopy.paragraph, inlineBlockCopy.button %}
 
-<div class="ama__inline-block-copy">
+<div class="ama__inline-block-copy col-sm-12 col-md-12 col-lg-4">
```
`col-md-12` and `col-sm-12` are still setting this element to `width: 100%` and it doesn't appear that `col-lg-4` is having any effect at all. I would recommend handling the sizing directly within the `scss` for `ama__inline-block-copy`

designerneil replied:
@labbydev perhaps I'm confused how this should be viewed on mobile devices but I thought we wanted the inline block to render at 100% on tablet and mobile and around 25% on desktop. If the grid sizing is not working then we will need to revisit it right away.

Commit:
Merge pull request #67 from AmericanMedicalAssociation/bugfix/EWL-4424-Create-News-Article-Template

EWL-4424 Adds link to body copy

Commit:
Merge pull request #63 from AmericanMedicalAssociation/feature/EWL-4452-member-pitch

EWL-4452 member pitch

designerneil commented:
## **Mobile**
<img width="510" alt="screen shot 2018-01-31 at 4 10 32 pm" src="https://user-images.githubusercontent.com/2271747/35650330-4ee6125e-06a1-11e8-9522-107de925b1a7.png">

## **Tablet**
<img width="1012" alt="screen shot 2018-01-31 at 4 10 24 pm" src="https://user-images.githubusercontent.com/2271747/35650331-4ef078f2-06a1-11e8-8b8f-c4c12ddf7771.png">

## **Desktop**
<img width="1261" alt="screen shot 2018-01-31 at 4 10 15 pm" src="https://user-images.githubusercontent.com/2271747/35650332-4f000dbc-06a1-11e8-97b4-7e73cc0a606c.png">

designerneil committed:
Merge branch 'bugfix/EWL-4429-promo-as-inline' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into bugfix/EWL-4429-promo-as-inline

designerneil committed:
Merge pull request #62 from AmericanMedicalAssociation/bugfix/EWL-4431-membership-promo-on-news-articles

refs EWL-4431: set font weight to 400, and update button colors

designerneil committed:
Changes the breakpoints do the inline block doesn't go full width on tablet.

designerneil committed:
Changes the breakpoints do the inline block doesn't go full width on tablet.

designerneil commented:
@labbydev the inline block will remain at 25% until the viewport reaches mobile size. Thanks again

labbydev commented:
@designerneil now the inline promo is working properly but the sidebar is broken now.

![localhost_3000_patterns_05-pages-news_05-pages-news html](https://user-images.githubusercontent.com/1653723/35700808-a78f9df4-0759-11e8-8623-770f06812164.jpg)

designerneil committed:
Fixes rebasing issues which caused a regression.
the columns no longer worked correctly

designerneil committed:
Fixes rebasing issues which caused a regression.
the columns no longer worked correctly

designerneil commented:
@labbydev sorry about the column thing. I was introduced right before I pushed my branch when I was rebasing it off of develop.

designerneil committed:
Merge branch 'bugfix/EWL-4429-promo-as-inline' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into bugfix/EWL-4429-promo-as-inline
